packages/react-select/src/types.ts

```diff
@@ -160,14 +161,20 @@ export interface CreateOptionActionMeta<Option extends OptionBase>
   action: 'create-option';
   name?: string;
 }
+export interface InitialInputFocusedActionMeta<Option extends OptionBase>
+  extends ActionMetaBase<Option> {
+  action: 'initial-input-focus';
+  name?: string;
+}
 
 export type ActionMeta<Option extends OptionBase> =
   | SelectOptionActionMeta<Option>
   | DeselectOptionActionMeta<Option>
   | RemoveValueActionMeta<Option>
   | PopValueActionMeta<Option>
   | ClearActionMeta<Option>
-  | CreateOptionActionMeta<Option>;
+  | CreateOptionActionMeta<Option>
+  | InitialInputFocusedActionMeta<Option>;
```

Comment:
I don't think we should be adding `InitialInputFocusedActionMeta` to `ActionMeta`, because that implies that the `onChange` prop can be called with `InitialInputFocusedActionMeta` for its action meta and that the user needs to handle it. However at the moment `InitialInputFocusedActionMeta` is only used for aria selection.

Reply:
I removed this from the ActionMeta type and used the ```InitialInputFocusedActionMeta``` interface directly in the ```AriaSelection``` type